fvm/evm/evm_test.go

```diff
@@ -1314,8 +1314,217 @@ func TestCadenceOwnedAccountFunctionalities(t *testing.T) {
 
 				transaction() {
 					prepare(account: auth(BorrowValue) &Account) {
-						let admin = account.storage
-							.borrow<&FlowToken.Administrator>(from: /storage/flowTokenAdmin)!
+						let admin = account.storage.borrow<&FlowToken.Administrator>(
+							from: /storage/flowTokenAdmin
+						)!
+
+						let minter <- admin.createNewMinter(allowedAmount: 2.34)
+						let vault <- minter.mintTokens(amount: 2.34)
+						destroy minter
+
+						let cadenceOwnedAccount <- EVM.createCadenceOwnedAccount()
+						cadenceOwnedAccount.deposit(from: <-vault)
+
+						let bal = EVM.Balance(attoflow: 9999999999)
```

Comment:
👍 
```suggestion
						// since 1e10 attoFlow is the minimum withdraw amount,
						// verify any amount below 1e10 can not be withdrew.
						let bal = EVM.Balance(attoflow: 9999999999)
```

Reply:
Updated in https://github.com/onflow/flow-go/pull/6877/commits/9a082871941581bfd4a14f44647a24ae059e03b0 .